src/baklava/input/IntegerInc.vue

```diff
@@ -60,7 +95,28 @@ export default class IntegerInc extends Vue {
     padding: 0 0.3em 0 0.1em;
   }
 
-  .inc-button:hover {
+  .inc-button:hover, .text-display:hover, input:hover {
     background: #e0e0e0e0;
   }
+
+  input {
+    width: 20px;
```

Comment:
maybe we could make this a responsive width. right now the box can only display an integer made up of maximum 3 digits. Would be nice to expand the box to fit the number?

Reply:
Yeah alright, I'll take a look

Reply:
I could only do it through TypeScript (it's not possible through CSS), but it works now.
![image](https://user-images.githubusercontent.com/47478125/96319966-fddf8880-1008-11eb-92a7-f4efbcd03a34.png)

Reply:
I just tested this and the box goes over the width of the node for very long numbers - it would be nice if we ensured the box had a max width. Also the box resizes when you click out of it and not as you type so you cannot actually see the whole number you have typed until you enter it. 

Would be interested to know your thoughts on these?

Reply:
1. My thoughts were that we'd be rewriting nodes anyway, and maybe we'd scale the width of the node to its contents.
2. A lot of work for something I didn't think was too important. It's just a tonne of unnecessary Javascript imo.

Reply:
And also, I would think that people aren't often going to want immensley large numbers anyway.

Reply:
1. I am not quite sure how this relates to rewriting nodes. Can we use the max width property to set it to the parent components width? 
2. I definately think this is important for providing a good user experience. Maybe create a ticket in our backlog so we can track this.

Reply:
1. We should rewrite the CSS for the node's width, otherwise if we have an option with a long name, then the maximum number you could put in could be only 2 digits or something. I would say it's a problem to be solved with the node, not the input.
2. It's hardly a bad user experience, I really don't see why people would be typing in enormous numbers. As soon as they press enter, the whole number is visible anyway. If it's something you think is important, you can add it to the backlog.

Reply:
1. Okay you have conviced me! 
2. Will do :smile: